content/docs/getting-started.md

```diff
@@ -18,109 +18,110 @@ redirect_from:
 
-### Glossary and FAQ {#glossary-and-faq}
+### Terimler sözlüğü ve SSS {#glossary-and-faq}
 
-The [glossary](/docs/glossary.html) contains an overview of the most common terms you'll see in the React documentation. There is also a FAQ section dedicated to short questions and answers about common topics, including [making AJAX requests](/docs/faq-ajax.html), [component state](/docs/faq-state.html), and [file structure](/docs/faq-structure.html).
+[Terimler sözlüğü](/docs/glossary.html) React anlatımlarında göreceğiniz en yaygın terimlerin bir özetini içerir. [AJAX isteği yapma](/docs/faq-ajax.html), [bileşen durumu](/docs/faq-state.html), and [dosya yapısı](/docs/faq-structure.html) dahil olmak üzere ortak konular hakkında kısa soru ve cevaplara ayrılmış bir SSS bölümü de bulunmaktadır.
 
-## Staying Informed {#staying-informed}
+## Haberdar olun {#staying-informed}
 
-The [React blog](/blog/) is the official source for the updates from the React team. Anything important, including release notes or deprecation notices, will be posted there first.
+[React blog](/blog/) React ekibinden yapılan güncellemelerin resmi kaynağıdır. Sürüm notları veya itiraz bildirimleri dahil olmak üzere, önemli olan her şey önce buraya gönderilecektir.
 
-You can also follow the [@reactjs account](https://twitter.com/reactjs) on Twitter, but you won't miss anything essential if you only read the blog.
+Twitter'da [@reactjs hesabını](https://twitter.com/reactjs) da takip edebilirsiniz, ama sadece blogu okursanız önemli hiç bir şeyi kaçırmazsınız.
 
-Not every React release deserves its own blog post, but you can find a detailed changelog for every release [in the `CHANGELOG.md` file in the React repository](https://github.com/facebook/react/blob/master/CHANGELOG.md), as well as on the [Releases](https://github.com/facebook/react) page.
+Her React sürümü kendi blog gönderisini hak etmiyor ama her sürüm için yapılan değişikliklerin ayrıntıarını [React havuzunun içindeki `CHANGELOG.md` dosyasında](https://github.com/facebook/react/blob/master/CHANGELOG.md) ve [Sürümler](https://github.com/facebook/react) sayfasında bulabilirsiniz.
 
-## Versioned Documentation {#versioned-documentation}
+## Versiyon Anlatımı {#versioned-documentation}
 
-This documentation always reflects the latest stable version of React. Since React 16, you can find older versions of the documentation [on a separate page](/versions). Note that documentation for past versions is snapshotted at the time of the release, and isn't being continuously updated.
+Bu bölüm her zaman React'ın en son kararlı versiyonunu yansıtır. React 16'dan beri, açıklamaların eski sürümlerini [ayrı bir sayfada](/versions) bulabilirsiniz. Geçmiş sürümlere ait açıklamaların yayınlanma tarihinde anlık olarak görüntülendiğini ve sürekli güncellenmediğini unutmayın.
```

Comment:
React'in daha uygundur.

Reply:
Can you also clarify this,, do you suggest I use 'React'in' instead of 'React'ın'?

Reply:
I suggest `React'in` instead of 'React'ın'. Beacuse people read this foreign language words as orginal. You can check [this documentation](https://turkofoni.org/files/yabanci_k_kenl__b_l___m_ter_mler_n_n_yazili_lari-prof.dr._haluk_akalin.htm) about apostrope using for foreign words in Turkish language.